src/tools/dart-create.md

```diff
@@ -0,0 +1,127 @@
+---
+title: dart create
+description: Command-line tool for creating Dart projects.
+toc: false
+---
+
+The `dart create` command allows creating Dart projects
+from varying available templates through a command line
+or many IDE project creation workflows.
+
+{% include tools/dart-tool-note.md %}
+
+The command creates a directory with the specified name, 
+creates necessary files, configures static analysis, 
+and retrieves any necessary dependencies.
+
+```terminal
+$ dart create [ARGUMENTS] <DIRECTORY>
+```
+
+{{site.alert.note}}
+  The command will attempt to run [`dart pub get`](/tools/pub/cmd/pub-get) 
+  after the project is created.
+
+  To avoid this you can specify the `--no-pub` flag:
+
+  ```terminal
+  $ dart create --no-pub <DIRECTORY>
+  ```
+{{site.alert.end}}
+
+The file structure created and the dependencies specified
+are configured through a choice of templates. By default, 
+the `console-simple` template is used, 
+creating a simple command-line application.
+
+To use that default template and create an application with
+the package and directory name `my_cli`, run the following command:
+
+```terminal
+$ dart create my_cli
+```
+
+This results in a package structure similar to the following:
+
+```bash
+my_cli/                  # The projects parent directory
+  bin/                   # The bin directory for the CLI tools
+  .gitignore             # Prevents various Dart files from being added to Git staging
+  analysis_options.yaml  # Configures static analysis for the project
+  CHANGELOG.md           # Allows you to specify changes between releases
+  pubspec.lock           # Pins down the dependency information for your package
+  pubspec.yaml           # Configures package information and dependencies
+  README.md              # Provides a description for your application
+```
```

Comment:
Do you think it's worth having an example outlining a created file structure like this? With the comments and everything?

Reply:
I'd prefer to have a link or a shared graphic/text. That makes it less likely that the info will get out of date.

Reply:
Good point, I've instead consolidated this part to a link to existing package conventions. I wasn't so confident on the wording though, so open to suggestions. Thanks!